themes/default/content/blog/cloud-systems-part-four/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Cloud Systems 101: Part Four"
+
+date: 2022-09-23T15:29:57-05:00
+
+meta_desc: In this series, learn modern cloud engineering practices and tooling, continuing with deploying our containerized website to Amazon's Elastic Kuberetes Service!
+
+meta_image: meta.png
+
+authors:
+    - kat-cosgrove
+
+tags:
+    - cloud-systems
+    - tutorials
+    - kubernetes
+
+---
+
+Cloud engineering is taking over software development. In a lot of ways, this is great; it allows us to build and deploy more complicated applications with less difficulty, and maintaining those applications becomes less troublesome too. We can release smaller updates more quickly than ever, ensuring that we can stay on top of feature requests and security issues. That said, the rise of cloud engineering has also introduced a lot of complexity in the form of dozens of services even within just one cloud provider. Figuring out where to start can be tough, so let’s take a practical tour! In this series, I’ll walk you through building a personal website and deploying it using modern cloud engineering practices.
+
+<!--more-->
+
+## Elastic Kubernetes Service
+
+In the previous tutorial, we extended our personal website to use the Flask web framework, add server-side routing, and package everything up into a Docker container. We then used Pulumi to deploy it to AWS Elastic Container Service. What if we want to go bigger, though? Maybe we're expecting a ton of traffic to the site. Let's make it a little more resilient by upgrading from Elastic Container Service to Kubernetes.
+
+If you aren't familiar with Kubernetes, it's a container orchestration tool. Tools like this are responsible for the management, deployment, and scaling of containerized applications. It's possible to do all of that by hand, of course, but it's slow, difficult, and unreliable. If you're just now joining me, you can get the completed code by forking and cloning [this repository](https://github.com/katcosgrove/cloud-systems-101).
+
+## Prerequisites:
+
+- An AWS Account
+
+- [Pulumi account](https://app.pulumi.com)
```

Comment:
We seem to get a lot of comments around not having info on installing the CLI and people having trouble finding the link. Maybe add that here like this?

```suggestion
- [Pulumi account](https://app.pulumi.com)
- [Pulumi CLI]({{< relref "/docs/get-started/install" >}})
```